tests/goth_tests/test_mid_agreement_payments/test_mid_agreement_payments.py

```diff
@@ -47,24 +56,24 @@ async def test_mid_agreement_payments(
     config_overrides: List[Override],
     single_node_override: Override,
 ) -> None:
-
+    # goth setup
     goth_config = load_yaml(goth_config_path, config_overrides + [single_node_override])
-    test_script_path = str(Path(__file__).parent / "requestor.py")
-
     configure_logging(log_dir)
-
-    runner = Runner(base_log_dir=log_dir, compose_config=goth_config.compose_config)
-
-    async with runner(goth_config.containers):
-
-        requestor = runner.get_probes(probe_type=RequestorProbe)[0]
-
-        async with requestor.run_command_on_host(test_script_path, env=os.environ) as (
-            _cmd_task,
+    goth_runner = Runner(base_log_dir=log_dir, compose_config=goth_config.compose_config)
+    async with goth_runner(goth_config.containers):
+        # given
+        given_requestor = goth_runner.get_probes(probe_type=RequestorProbe)[0]
```

Comment:
overall, I can say that I _generally_ don't like the changes to the variable names in this file... the variables were more or less sufficiently readable as they were and in my opinion, your use of longer names _does_ not translate to better code readability but rather _increases_ the attention level required to read it... ... 

plus, I really don't understand the prefix `given_` here... the requestor is not some precondition... it's just a node in the network...